### ISSUE TITLE
the param of cnn

### ISSUE THREAD

**Issue by HappyLuffy** [gramuah/ccnn#1]
Hello, the params for CCNN are as below
![ccnn](https://user-images.githubusercontent.com/27480099/28574890-d7a559de-7181-11e7-83e1-19e71eeaa25e.png)
But I think the Conv2 should be 36\*36\*32 and Conv3 should be 18\*18\*64.
I want to know if I had made any misunderstanding about the paper. Thank you.

**Comment by roleiland** [comment]
Yes, you are right. It should be 36*36*62. It is wrong in the figure.

**Comment by HappyLuffy** [comment]
Thank you for your answer.
